src/overlays/actors/ovl_En_Warp_tag/z_en_warp_tag.c

```diff
@@ -209,13 +209,13 @@ void EnWarpTag_RespawnPlayer(EnWarptag* this, PlayState* play) {
                 // why are we getting player home rotation from the room data? doesnt player have home.rot.y?
                 // especially because we are converting from deg to binang, but isnt home.rot.y already in binang??
                 Play_SetRespawnData(
-                    &play->state, 0, entrance, // parameter 3 is called "sceneSetup"
-                    play->setupEntranceList[playerSpawnIndex].room, playerParams, &newRespawnPos,
+                    &play->state, 0, entrance, play->setupEntranceList[playerSpawnIndex].room, playerParams,
+                    &newRespawnPos,
                     ((((playerActorEntry->rot.y >> 7) & 0x1FF) / 180.0f) * 32768.0f)); // DEG_TO_BINANG ?
 
-                func_80169EFC(&play->state);
+                Play_TriggerVoidOut(&play->state);
                 gSaveContext.respawnFlag = ~0x4;
-                func_80165690();
+                Play_DisableMotionBlur();
```

Comment:
There's probably quite a few functions that can be added to namefixer.py. Might even be worth including the `TRANS_TYPE_` and `TRANS_MODE_` enums in there